controller/flight_handlers.go

```diff
@@ -0,0 +1,54 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/cyneptic/letsgo/controller/validators"
+	"github.com/cyneptic/letsgo/infrastructure/provider"
+	"github.com/cyneptic/letsgo/internal/core/entities"
+	"github.com/cyneptic/letsgo/internal/core/ports"
+	"github.com/cyneptic/letsgo/internal/core/service"
+	"github.com/labstack/echo/v4"
+)
+
+type FlightHandler struct {
+	svc ports.FlightServiceContract
+}
+
+func NewFlightHandler(svc ports.FlightServiceContract) *FlightHandler {
+	return &FlightHandler{
+		svc: svc,
+	}
+}
+
+func AddFlightRoutes(e *echo.Echo) {
+	pv := provider.NewFlightProviderClient()
+	svc := service.NewFlightService(pv)
+	handler := NewFlightHandler(svc)
+	e.GET("/flights", handler.ListFlightsHandler)
+	e.GET("/flights/:id", handler.flightHandler)
```

Comment:
You are returning one flight, using the word 'flight(s)' here can be misdirecting, use flight/id: instead.